packages/edit-site/src/components/sidebar-edit-mode/template-panel/index.js

```diff
@@ -31,10 +33,14 @@ export default function TemplatePanel() {
 		const postType = getEditedPostType();
 		const postId = getEditedPostId();
 		const _record = getEditedEntityRecord( 'postType', postType, postId );
+		const info = getTemplateInfo( _record );
 
-		const info = _record ? getTemplateInfo( _record ) : {};
-
-		return { info, record: _record };
+		return {
+			title: info.title,
+			description: info.description,
+			icon: info.icon,
+			record: _record,
+		};
```

Comment:
I've refactored the return values a bit. The `info` will never have a stable reference, so it's better to return values directly.